server/src/controllers/api/projects_controller.js

```diff
@@ -117,11 +118,22 @@ async function getProjects(req, res) {
       $sort: sortParams,
     },
     {
-      $limit: pageSize,
+      $facet: {
+        metaData: [
+          { $count: 'total' },
+        ],
+        data: [
+          { $skip: (page - 1) * pageSize },
+          { $limit: pageSize },
+        ],
+      },
     },
   ])
     .exec();
-  const projects = await ProjectOwner.populate(aggrProjects, { path: 'projectOwner' });
+
+  const populatedProjects = aggrProjects[0].data;
+  const projects = await ProjectOwner.populate(populatedProjects, { path: 'projectOwner' });
+  // const totalDocs = aggrProjects[0].metaData[0].total;
```

Comment:
Since we are getting the total count from the call to `/projects/counts` then we don't need this and we can remove it